docs/updating/upgrading_to_v3.md

```diff
@@ -26,10 +26,23 @@ This folder also contains `config/packages/ezplatform.yaml`, which contains all
 
 #### PHP code and bundle organization
 
-Since Symfony 4 your code is no longer organized in bundles.
-`AppBundle` has been removed from eZ Platform.
-You need to move all your PHP code, such as controllers or event listeners, to the `src` folder.
-Use the `App` namespace for your custom code instead.
+Since Symfony 4 `src/` code is no longer organized in bundles, `AppBundle` has been removed from the default eZ Platform install.
+In order to adapt, you'll need to move all your PHP code, such as controllers or event listeners, to the `src` folder and use the `App` namespace for your custom code instead.
+
+!!! tip "How to make AppBundle continue to work, for now"
+
+    Refactoring bundles for `src/` folder can involve extensive changes, if you want to make your `src/AppBundle` continue to work, follow [an Autoloading src/ & src/AppBundle guide on Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle).
```

Comment:
```suggestion
    Refactoring bundles for `src/` folder can involve extensive changes, if you want to make your `src/AppBundle` continue to work, follow [an Autoloading src/AppBundle guide on Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle).
```

@MagdalenaZuba Afaik `src/` is already autoloaded by default in eZ Platform, so can simplify text here.
See: https://github.com/ezsystems/ezplatform/blob/master/composer.json#L104-L113

Reply:
I just copied the title from the link, so the user will know from the start what to expect.

Reply:
but I like the simplified version better, so yes, why not :)